sugoi-api-distribution/sugoi-api-distribution-jar/src/main/resources/application.properties

```diff
@@ -100,4 +101,30 @@ sugoi.api.event.webhook.web1.tag=MAIL
 #can also be an url
 sugoi.api.event.webhook.web1.default.reset.template=classpath:template/mail_default.ftl
 #can also be an url
-sugoi.api.event.webhook.web1.default.init.template=classpath:template/mail_default.ftl
\ No newline at end of file
+sugoi.api.event.webhook.web1.default.init.template=classpath:template/mail_default.ftl
+
+
+## ACTUATOR AND METRICS
+management.endpoints.web.exposure.include=*
+management.health.defaults.enabled=true
+management.health.jms.enabled=false
+management.health.ldap.enabled=false
+
+management.endpoint.health.show-details=when-authorized
+management.info.defaults.enabled=true
+management.endpoint.metrics.enabled=true
+management.endpoint.prometheus.enabled=true
+management.metrics.export.prometheus.enabled=true
+
+
+info.name=Sugoi API
+info.description=User management API
+info.version=${java.version}
```

Comment:
why not put ${project.version} here ? or nothing.

Reply:
put project.version here is not possible, because project.version is not injected, if we want to use project.vversion it's only available under info.build.version. I will remove info.version